code/game/objects/items/binoculars.dm

```diff
@@ -224,13 +241,15 @@
 					QDEL_NULL(laser)
 					break
 		if(MODE_RANGE_FINDER)
-			if(!linked_mortar)
-				to_chat(user, span_notice("No linked mortar found."))
+			if(!linked_mortars.len)
+				to_chat(user, span_notice("No linked mortars found."))
 				return
+			check_mortar_index() // incase varedit screws something up
 			targetturf = TU
-			to_chat(user, span_notice("COORDINATES TARGETED: LONGITUDE [targetturf.x]. LATITUDE [targetturf.y]."))
+			to_chat(user, span_notice("COORDINATES TARGETED BY MORTAR [selected_mortar]: LONGITUDE [targetturf.x]. LATITUDE [targetturf.y]."))
 			playsound(src, 'sound/effects/binoctarget.ogg', 35)
-			linked_mortar.recieve_target(TU,user)
+			var/obj/machinery/deployable/mortar/mortar = linked_mortars[selected_mortar] /// typecasted lists STILL dont help in the slightest
```

Comment:
Remove comment

Also typecasting list is just for readability

Reply:
done